umbral/utils.py

```diff
@@ -42,3 +44,23 @@ def kdf(ecpoint, key_length):
 
 def get_curve_keysize_bytes(curve):
     return int(math.ceil(curve.key_size / 8.00))
+
+
+class AbstractFromToBytes(ABC):
+    @abstractmethod
+    @staticmethod
+    def from_bytes(cls, data: bytes, curve: ec.EllipticCurve = None):
+        """
+        Instantiate a KFrag object from the serialized data.
+        """
+        pass
+
+    @abstractmethod
+    def to_bytes(self):
+        """
+        Serialize the KFrag into a bytestring.
+        """
+        pass
```

Comment:
Hmm, as per the official python docs, this function needs to raise `NotImplementedError`.  See: https://docs.python.org/3/library/exceptions.html#NotImplementedError

Reply:
thanks! I didn't know about this